components/com_fabrik/models/list.php

```diff
@@ -11984,6 +11985,17 @@ private function getTabCategories()
 		JDEBUG ? $profiler->mark('after fabrik list tabs query run') : null;
 		FabrikHelperHTML::debug($counts, 'list getTabCategories counts: ' . $table->label);
 
+		if ($isDropdown) {
+			/* Convert element values to names */
+			foreach ($counts as $key => $count) {
+				$counts[$key][0] = $elementModel->getLabelForValue($counts[$key][0]);
+			}
+			/* Now sort it by these labels */
+			usort($counts, function($a, $b) {
```

Comment:
Unclear why you are using `usort` rather than simpler `sort`?

Reply:
multidimensional array.

Reply:
array_multisort?